cart.spec

```diff
@@ -0,0 +1,82 @@
+Name:          cart
+Version:       0.0.1
+Release:       1%{?dist}
+Summary:       CaRT
+
+License:       Apache
+URL:           https//github.com/daos-stack/cart
+Source0:       %{name}-%{version}.tar.gz
+Source1:       scons_local-%{version}.tar.gz
+
+BuildRequires: scons
+BuildRequires: libfabric-devel
+BuildRequires: pmix-devel
+BuildRequires: openpa-devel
+BuildRequires: mercury-devel
+BuildRequires: ompi-devel
+BuildRequires: libevent-devel
+BuildRequires: boost-devel
+BuildRequires: libuuid-devel
+BuildRequires: hwloc-devel
+BuildRequires: openssl-devel
+BuildRequires: libcmocka-devel
+BuildRequires: libyaml-devel
+#Requires:
+
+%description
+
+Collective and RPC Transport (CaRT)
+
+CaRT is an open-source RPC transport layer for Big Data and Exascale
+HPC. It supports both traditional P2P RPC delivering and collective RPC
+which invokes the RPC at a group of target servers with a scalable
+tree-based message propagating.
+
+%prep
+%setup -q
+%setup -q -a 1
+
+
+%build
+# remove rpathing from the build
+find . -name SConscript | xargs sed -i -e '/AppendUnique(RPATH=.*)/d'
+
+scons %{?_smp_mflags}                     \
+      --config=force                      \
+      --update-prereq=all                 \
```

Comment:
Remind me what this option does and why it's needed.

Reply:
I'm not entirely sure TBH.  It's just one of the number of "standard" options that we currently use successfully to build DAOS and CaRT with `scons` so I'd like to keep it consistent.

Reply:
It's a trade-off between flexibility and consistency, for RPMs I'd personally rather it was tightly defined and we had to fix things when it breaks, as at least then we know when something changes.

Reply:
Indeed.  Part of my next patch.